e2etest/native_staking_test.go

```diff
@@ -56,8 +56,9 @@ var (
 )
 
 var (
-	gasPrice = big.NewInt(0)
-	gasLimit = uint64(10000000)
+	gasPrice     = big.NewInt(0)
+	gasPrice1559 = big.NewInt(unit.Qev)
```

Comment:
move to `e2etest/contract_staking_v2_test.go` above, it is not used in this file